utils_bsd.go

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//go:build darwin || dragonfly || freebsd || netbsd || openbsd
```

Comment:
isn't darwin mac?

Reply:
yes it is
that's how golang names MacOS
> To build a file only when using cgo, and only on Linux and OS X:
>
> //go:build cgo && (linux || darwin)

Source: https://pkg.go.dev/cmd/go#hdr-Build_constraints

Reply:
ahh I see, darwin is bsd variant, I forgot :sweat_smile: